style.css

```diff
@@ -0,0 +1,81 @@
+body {
+  background-color: #f7f7f9;
+}
+
+header {
+  justify-content: space-between;
+  display: flex;
+  height: 48px;
+}
+
+header a {
+  font-family: "Poppins", Helvetica, sans-serif;
+  font-size: 18px;
+  font-weight: 700;
+  color: #6070ff;
+  padding: 12px 16px;
+  text-decoration: none;
+  display: flex;
+  align-items: center;
+}
+
+.headline {
+  display: flex;
+  flex-direction: column;
+  height: 100vh;
+  padding: 50px 20px;
+  background-image: url(images/header-shapes\ mobile.svg);
+  background-repeat: no-repeat;
+  background-position: top;
+}
+
+h2 {
+  font-family: "poppins", Helvetica, sans-serif;
+  font-weight: 700;
+  font-size: 48px;
+  color: #172b4d;
+  line-height: 52px;
+}
```

Comment:
- [ ] I think you forgot to change the CSS selector from h2 to h1 after making the changes in the HTML. This caused a small issue. Take a look at the pictures 
![image](https://user-images.githubusercontent.com/43698511/129275401-74395c71-89dd-4502-ba04-1e3674f55dfd.png)

![image](https://user-images.githubusercontent.com/43698511/129275430-2cea1307-5c51-4682-9c8f-68e40c391b1f.png)

Kindly fix the problem